pkg/api/handlers/libpod/pods.go

```diff
@@ -530,8 +530,12 @@ func PodStats(w http.ResponseWriter, r *http.Request) {
 	query := struct {
 		NamesOrIDs []string `schema:"namesOrIDs"`
 		All        bool     `schema:"all"`
+		Stream     bool     `schema:"stream"`
+		Delay      int      `schema:"delay"`
```

Comment:
is this exposed natively? if it is not, it should be added as a flag if applicable.

Reply:
What do you mean natively? I added this to docs and made it available in rest api

Reply:
see comment below.